buildzthinrpm_rhel

```diff
@@ -26,15 +26,15 @@ fi
 
 BRANCH=$1
 BUILD=/root/zthin-build
-REPO=https://github.com/mfcloud/python-zvm-sdk.git
-REPOTOP=python-zvm-sdk
+REPO=https://github.com/openmainframeproject/feilong
+REPOTOP=feilong
 ZTHINTOP=zthin-parts
 
 # Check whether branch exists or not
 EXISTS=$(git ls-remote -h $REPO | grep "$BRANCH"$ | wc -l )
 if [ "$EXISTS" == 0 ] ; then
    {
-   echo "Branch $BRANCH does not exist on /mfcloud/python-zvm-sdk"
+   echo "Branch $BRANCH does not exist on /openmainframeproject/feilong"
```

Comment:
echo "Branch $BRANCH does not exist on openmainframeproject/feilong"

Reply:
Done

Reply:
I meant, remove the `/` before `openmainframeproject/feilong`